beacon-chain/node/config.go

```diff
@@ -65,6 +65,24 @@ func configureSafeSlotsToImportOptimistically(cliCtx *cli.Context) error {
 	return nil
 }
 
+func configureMevBoostCircuitBreakerValues(cliCtx *cli.Context) error {
+	if cliCtx.IsSet(flags.BuilderFallbackSkips.Name) {
+		c := params.BeaconConfig().Copy()
+		c.BuilderFallbackSkipsSlot = types.Slot(cliCtx.Int(flags.BuilderFallbackSkips.Name))
```

Comment:
maybe rename to `MaxBuilderConsecutiveMissedSlots` telling the system that there should be a max of x slots missed before falling back to something